docs/user-guide/cli-authentication-methods.md

```diff
@@ -0,0 +1,288 @@
+# Zowe CLI authentication methods
+
+Zowe CLI supports multiple authentication methods including basic authentication, tokens for single sign-on, client certificates, and multi-factor authentication. 
+
+Configure the authentication method you want to use across multiple mainframe services in a [base profile](../appendix/zowe-glossary.md#base-profile) stored in your configuration file. 
+
+:::note Notes
+
+- To apply your authentication method to a specific service, define the authentication type in the [service profile](../appendix/zowe-glossary.md#service-profile) of your choice.
+
+- If you switch to a different authentication type, remove the previous authentication method from the profile it was stored in before.
+:::
+
+## Using basic authentication
+
+The advantage of basic authentication is the simplicity of set up: In most cases you can use Zowe CLI (or Zowe Explorer) to connect to mainframe services without additional configuration on the server. Other authentication methods would likely require further configuration.
+
+Basic authentication is the default authentication method defined in the default base profile when you issue the `zowe config init` command in Zowe CLI to create your team configuration.
+
+If you change your authentication method and want to switch back to using basic authentication:
+
+1. Use a text editor to update the `zowe.config.json` file to define the authentication as basic authentication (see the highlighted lines) in the base profile:
+   ```json
+         "project_base": {
```

Comment:
`project_base` is the default base profile name for a project team config but it could be different. Should we have a note mentioning that the default name is "global_base" for a global team config? The name could be anything if the base profile was renamed - if unsure the default base profile name can be found by running `zowe config list defaults`